extensions/sonarqube/overview.md

```diff
@@ -1,8 +1,9 @@
-**[SonarQube][sq]** is the leading product for Continuous Code Quality. It is an open source, static source code analysis solution that enables continuous tracking of bugs, code smells and vulnerabilities for more than 20 different languages such as C#, VB .Net, Java, C, C++, COBOL, JavaScript, ...
+**[SonarQube][sq]** is the leading product for Continuous Code Quality. It supports all major programming languages, including C#, VB .Net, JavaScript, TypeScript, C/C++ and many more. Integrate it in your on-premise TFS installation, and continuously track down bugs and vulnerabilities in your codebase.
 
-SonarQube can be [installed][getstarted] and run on a dedicated infrastructure.
+If you are new to SonarQube, using VSTS or looking for cloud-based solutions, then make sure to also check our dedicated [SonarCloud extension for VSTS][sc_vsts] !
+
+This TFS extension provides build tasks that you can add in your build definition. All branches and pull-requests are automatically analyzed, allowing you to discover early any bug or vulnerability in the code.
```

Comment:
This 'This TFS extension' seems weird! AFAIK the SonarCloud task is only available for VSTS and the SonarQube one available for both.

Reply:
Disregard my comment I misread the name of the md file.